Core/HLE/sceIo.cpp

```diff
@@ -317,8 +317,6 @@ static void __IoFreeFd(int fd, u32 &error) {
 // TODO: We don't do any of that yet.
 // For now, let's at least delay the callback notification.
 static void __IoAsyncNotify(u64 userdata, int cyclesLate) {
-	PROFILE_THIS_SCOPE("io_rw");
```

Comment:
Had to remove it because our profiler can't handle entering the same scope recursively.

However shouldn't have done it in this branch, and should probably rather have fixed the profiler :)